.github/workflows/main.yml

```diff
@@ -155,6 +163,9 @@ jobs:
     - uses: actions/checkout@v4
     - name: Dump Environment
       run: ci/dump-environment.sh
+    # workaround for https://github.com/actions/partner-runner-images/issues/26
```

Comment:
Based on the linked issue, it's available now.

Reply:
I re-ran the tests and everything passed except cross compile.

Reply:
The current arm machine sometimes throws a random `free()` allocation error. I lost the exact build failure, but feel like it is not really stable yet.

Reply:
Investigation into compilation problems are currently being investigated in https://github.com/rust-lang/rust/issues/135867.

Reply:
Found it <https://github.com/weihanglo/cargo/actions/runs/12939625507/job/36092284988#step:13:48>.

```
  free(): invalid next size (fast)
```

It doesn't seem to be a compilation error though.

Reply:
CI passed. Should we make it into our CI pipeline?